nbconvert/templates/latex/base.tplx

```diff
@@ -13,14 +13,14 @@ This template does not define a docclass, the inheriting class must define this.
     ((* block docclass *))((* endblock docclass *))
     
     ((* block packages *))
-	\usepackage{polyglossia}
-	\usepackage{fontspec}
+    \usepackage{polyglossia}
+    \usepackage{fontspec}
     \usepackage{mathtools} % Equations
     \usepackage{amssymb} % Equations
-	\usepackage{unicode-math}
-	\setmainfont{XITS}
-	\setmathfont{Latin Modern Math}
-	\setmathfont[range=\mathbb]{xits-math.otf}
```

Comment:
Somehow, the 4-spaces got converted into double-tabs, hence an error when travis checked.